AgentBlob/GenerateAgentBlob.ps1

```diff
@@ -0,0 +1,79 @@
+﻿
+#$BootstrapUrl = "http://52.151.39.54/performance"
+
+function Download-7Zip {
+    Write-output "Download-7Zip"
+    $urifile = "https://7-zip.org/a/7z1801-x64.msi"
+    $uri = [System.Uri]$urifile
+    $targetFilename = Split-Path -Path $uri.LocalPath -Leaf
+    write-host $targetFilename
+    Invoke-WebRequest -UseBasicParsing -Uri $urifile -OutFile .\7z1801-x64.msi
+}
+
+function Generate-WindowsAgentBlob {
+    Write-output "Generate-WindowsAgentBlob"
+    $BootstrapUrl = "http://dcos-win.westus.cloudapp.azure.com/dcos-windows/stable"
+    $TARGET_DIR = "C:\agentblob"
```

Comment:
Can we use `$env:WORKSPACE` (it's a predefined variable in Jenkins) as the working directory, instead of using `C:\` ?

We would simple change `$TARGET_DIR = "C:\agentblob"` to `$TARGET_DIR = Join-Path $env:WORKSPACE "agentblob"`

Reply:
I will address this.